WNPRC_EHR/test/src/org/labkey/test/tests/wnprc_ehr/WNPRC_EHRTest.java

```diff
@@ -898,6 +903,13 @@ private void enterCharges()
         clickAndWait(Locator.bodyLinkContainingText("Enter Charges with Animal Ids"));
         enterChargesInGrid(1, mapWithAnimalId);
 
+        log("Submit & Reload Form");
+        sleep(5000);
```

Comment:
@labkey-sweta There's a lot of sleeps() in here. Any suggestions on an alternative?

Reply:
Suggested this wait condition instead of sleep.
waitForElementToDisappear(Locator.tagWithClassContaining("div","tip-body-form-invalid"),5000);

Reply:
Thanks, @labkey-sweta! For this suggestion and other one as well. However, they did not work. I had tried various alternatives before resorting to sleep statements previously.